### ISSUE TITLE
Google Maps: make it apparent of waypoint amount limits

### ISSUE THREAD

**Issue by AlexGStapleton** [siteorigin/so-widgets-bundle#285]
Make it apparent that there is a limit of 8 waypoints for free users and 23 for paid users. Ideally, we should show this when the user reaches this limit rather than showing it all the time. [Reference](https://developers.google.com/maps/documentation/javascript/directions#UsageLimits)

**Comment by gregpriday** [comment]
Overly documenting these sorts of limitations inside a form can make them really messy. Ideally, we should just point the user to these limitations in the documentation.

**Comment by AlexGStapleton** [comment]
Hm, that's a good point. What about a (text) warning if the user inputs a value higher than the limited amount?

**Comment by braamgenis** [comment]
I like the idea of warning the user when they go over the limit. Will see if I can implement this easily.

**Comment by Misplon** [comment]
https://cloud.google.com/maps-platform/user-guide/pricing-changes/

_Directions API

Billing for the Directions API will change as follows:

The maximum allowed number of waypoints has been increased to 25. Previously, calls were limited to 10 or 23 waypoints. Contact sales if you need to access more than 25 waypoints.
Directions requests will be billed at a lower rate for calls with up to 10 waypoints and a higher rate for calls with 11–25 waypoints.
Directions Advanced requests using waypoint optimization and traffic models will also be billed at a higher rate.
See the new pricing table for details._

**tl;dr New limit is 25.**
